rules/network/command_and_control_ssh_secure_shell_to_the_internet.toml

```diff
@@ -32,7 +32,7 @@ tags = ["Elastic", "Network"]
 type = "query"
 
 query = '''
-network.transport:tcp and destination.port:22 and
+event.category:(network or network_traffic) and network.transport:tcp and (destination.port:22 or event.dataset:zeek.ssh) and
```

Comment:
Can you update the `updated_date` and `ecs_version` in `metadata` for this rule?